### PR TITLE
VZ-9468: Revert to using rancher/charts 

### PR DESCRIPTION
**Pull request by mgianatagh**
Revert to using rancher/charts instead of verrazzano/rancher-charts

### CODE REVIEW

**Comment by jmaron99** [comment]
Are we deleting the VZ rancher-charts repo?  I imagine the PLS is still required?

File: pkg/data/management/catalog_data.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	libraryURL    = "https://github.com/verrazzano/rancher-charts"
+	libraryURL    = "https://git.rancher.io/charts"
```
Comment (jmaron99) [inline pkg/data/management/catalog_data.go:27]: I'm confused about this libraryURL - it's unreachable from my host.  Perhaps it should be rancher/rancher-charts?

Comment (mgianatagh) [reply pkg/data/management/catalog_data.go:27]: I have reverted to this to how it is in the Rancher code.  It seems to be a helm repo that gets it's content from GitHub.com/rancher/charts

**Review by theabmitra — APPROVED** [review]
lgtm

**Review by Abhijithang — APPROVED** [review]

**Pull request merged**